krb5wrap.cpp

```diff
@@ -270,8 +273,7 @@ KRB5Keytab::cursor::~cursor()
                                               m_keytab.m_keytab,
                                               &m_cursor);
     if (ret) {
-        /* FIXME: shouldn't throw from destructor... */
-        throw KRB5Exception("krb5_kt_end_seq_get", ret);
+        krb5_error_exit("krb5_kt_end_seq_get", ret);
```

Comment:
Exception from `KRB5Keytab::cursor` are currently caught and ignored, so a forced exit changes (intended) semantics.